backend/project-management-system/src/main/java/org/egov/works/web/models/ProjectRequest.java

```diff
@@ -21,10 +21,10 @@
 @NoArgsConstructor
 @Builder
 public class ProjectRequest {
-    @JsonProperty("requestInfo")
+    @JsonProperty("RequestInfo")
     private RequestInfo requestInfo = null;
 
-    @JsonProperty("projects")
+    @JsonProperty("project")
```

Comment:
If it is an array of projects, the JSON property itself should be called "Projects". It translates to a list, right?

Reply:
Property renamed to Projects. Yes it translates to a list